optimade/validator/validator.py

```diff
@@ -452,11 +452,12 @@ def _get_archetypal_entry(
 
         """
         response, _ = self._get_endpoint(endp)
-        data_returned = response.json()["meta"]["data_returned"]
+        data = response.json().get("data", [])
```

Comment:
This method (`_get_archetypal_entry`) is only run for non-single entry endpoints?

Reply:
Correct